test/functional/pages_controller_test.rb

```diff
@@ -19,4 +19,11 @@ class PagesControllerTest < ActionController::TestCase
     get :index
     assert_nil assigns(:repos_subs), 'not assigns to repos_subs'
   end
+
+  test "cache control for headers" do
+    get :index
+    assert_equal(@response.headers["Cache-Control"], "no-cache, no-store, max-age=0, must-revalidate")
```

Comment:
Can you flip the arguments? the expected value should come first and then actual.